### ISSUE TITLE
Type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols

### ISSUE THREAD

**Issue by andreyz** [finestructure/PreviewProviderTools#9]
Following compilation errors prevent Gala 0.3.3 from building with Xcode 11.6:
```
Gala/Sources/Gala/Gala.swift:28:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:28:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:46:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:46:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([UserInterfaceSizeClass.compact, .regular], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:70:5: error: type '()' cannot conform to 'View'; only struct/enum/class types can conform to protocols
    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
    ^
Gala/Sources/Gala/Gala.swift:70:5: note: required by referencing initializer 'init(_:id:content:)' on 'ForEach' where 'Content' = '()'
    ForEach([LegibilityWeight.regular, .bold], id: \.self) { item in
    ^
```

**Comment by finestructure** [comment]
Ooops, looks like I broke 11.6 when I fixed the Xcode 12 beta. Thanks for reporting it!

**Comment by finestructure** [comment]
That should be fixed in 0.3.4, please let me know if you're still having problems!

**Comment by andreyz** [comment]
Works great in both Xcode 11.6 and 12 beta 4. Thank you for a useful library and a very quick patch!
